pkg/pillar/devicenetwork/dns.go

```diff
@@ -36,7 +36,7 @@ func GetDhcpInfo(us *types.NetworkPortStatus) error {
 		errStr := fmt.Sprintf("dhcpcd -U failed %s: %s",
 			string(stdoutStderr), err)
 		log.Errorln(errStr)
-		return nil
```

Comment:
Hmm.. Should this error not be returned?

Reply:
No that is noise; get those in the logs all the time when dhcpcd has stopped or is not running. Doesn't make the config invalid.